qt/widgets/instrumentview/src/InstrumentActor.cpp

```diff
@@ -1304,5 +1329,53 @@ void InstrumentActor::loadFromProject(const std::string &lines) {
   }
 }
 
-} // MantidWidgets
-} // MantidQt
+const Mantid::Geometry::ComponentInfo &InstrumentActor::componentInfo() const {
+  return getComponentInfo();
```

Comment:
Why are there `getComponentInfo` and `componentInfo` methods?. Surely only `componentInfo` is needed? Also, very important to comment that this will return EITHER the neutronic or physical version. Other similiar APIs (on `MatrixWorkspace` for example) only return the neutronic version.